packages/utils/test/string.test.ts

```diff
@@ -45,3 +49,60 @@ describe('isMatchingPattern()', () => {
     expect(isMatchingPattern([] as any, 'foo')).toEqual(false);
   });
 });
+
+// NOTE: These tests are copied (and adapted for chai syntax) to `string.test.ts` in `@sentry/browser`. The
+// base64-conversion functions have a different implementation in browser and node, so they're copied there to prove
+// they work in a real live browser. If you make changes here, make sure to also port them over to that copy.
+describe('base64ToUnicode/unicodeToBase64', () => {
+  const unicodeString = 'Dogs are great!';
+  const base64String = 'RG9ncyBhcmUgZ3JlYXQh';
+
+  test('converts to valid base64', () => {
+    expect(BASE64_REGEX.test(unicodeToBase64(unicodeString))).toBe(true);
+  });
+
+  test('works as expected', () => {
+    expect(unicodeToBase64(unicodeString)).toEqual(base64String);
+    expect(base64ToUnicode(base64String)).toEqual(unicodeString);
+  });
+
+  test('conversion functions are inverses', () => {
+    expect(base64ToUnicode(unicodeToBase64(unicodeString))).toEqual(unicodeString);
```

Comment:
This is logically implied from the assertion above it.

Reply:
I suppose that's true. I can remove it. (I'm not going to push the change until after the ember test problem is fixed, though, because I'm just going to have to rebase on top of that fix and push again at that point.)